student.rb

```diff
@@ -1,9 +1,12 @@
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(classroom, age, name = 'unknown', parent_permission: true)
     super(age, name, parent_permission: parent_permission)
     @classroom = classroom
+    classroom&.add_student(self)
```

Comment:
- Hey, here, we shouldn't have this line of code here, instead, we should create the missing method 
`def classroom=` which establishes the belongs relationship as a student belongs to a classroom. Kindly delete this line and add the method below:

```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```